docs/userGuide/componentsReference/searchbarDoc.md

```diff
@@ -0,0 +1,29 @@
+<frontmatter>
+footer: userGuideFooter.md
+siteNav: userGuideSections.md
+</frontmatter>
+
+<include src="../../common/header.md" />
+
+<div class="website-content">
+  <tip-box border-left-color="#00B0F0">
+    <i style="font-style: normal; font-weight: bold; color: dimgray">Example</i><br>
+    <searchbar :data="searchData" placeholder="Search" :on-hit="searchCallback"></searchbar>
+  </tip-box>
+
+  <tip-box border-left-color="black">
+  <i style="font-style: normal; font-weight: bold; color: dimgray">Markup</i>
+
+  ``` html
+  <searchbar :data="searchData" placeholder="Search" :on-hit="searchCallback"></searchbar>
+  ```
+  </tip-box>
+  <br>
```

Comment:
Can also add the Javascript code from the original documentation (the tip-box colors can be the same as the one used by Markup).